packages/compass-indexes/src/typings.d.ts

```diff
@@ -5,3 +5,6 @@ declare module 'mongodb-query-parser' {
   ) => false | null | CollationOptions;
   export { isCollationValid };
 }
+
+declare module 'hadron-react-buttons';
```

Comment:
It looks like we're removing the last reference of `hadron-react-buttons` in this package with these changes. Can we remove this line and the `hadron-react-buttons` dependency in this package?
```suggestion
```
I think there's one instance of `hadron-react-components` still used in `drop-index-modal`, however it's in a jsx file. Do we need the type declaration for that? Wondering if we can remove the line below as well.

Reply:
Removed `hadron-react-buttons`, but kept `hadron-react-components` because it is still in use.